HotelApp/Controllers/HotelController.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using HotelApp.API.DbContexts;
+using HotelApp.API.DbContexts.Entities;
+using HotelApp.API.DbContexts.Repositories;
+using HotelApp.API.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+
+namespace HotelApp.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class HotelController : ControllerBase
+    {
+        private readonly IHotelRepository _hotelRepository;
+        private readonly IHotelStatusRepository _hotelStatusRepository;
+        private readonly UserManager<User> _userManager;
+
+        public HotelController(IHotelRepository hotelRepository,
+                               IHotelStatusRepository hotelStatusRepository,
+                               UserManager<User> userManager)
+        {
+            _hotelRepository = hotelRepository;
+            _hotelStatusRepository = hotelStatusRepository;
+            _userManager = userManager;
+        }
+
+        [HttpPost]
+        [Authorize(Roles = "Registered user")]
+        [Route("register")]
```

Comment:
Ruta nek ti bude prazna. Registracija hotela je zapravo Create metoda i to po rest konvenciji ide HttpPost na /Hotel rutu